src/utils/__tests__/isKnownUnit-test.js

```diff
@@ -32,5 +33,20 @@ test("isKnownUnit", t => {
   t.ok(isKnownUnit("dpi"))
   t.ok(isKnownUnit("dpcm"))
   t.ok(isKnownUnit("dppx"))
+
+  t.ok(isKnownUnit("%", { only: "length" }))
```

Comment:
`only: "length"` should mean "is a known *length* unit". So `isKnownUnit("deg", { only: "length" })`, for example, should return `false`, right? Because it's not a known *length* unit?